CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Add custom state to context
 * Add `prng` function to utils
 * Add `choose` function to utils
+* Refactor to a reactive architecture
```

Comment:
README needs to be updated to reflect the new `placementEngine.subscribe` API, instead of `placementEngine.then` (Promise -> Bulb): https://github.com/conversaation/promos-client/tree/f4cd12506a6a26e2be28d227718dd85c2f165963#how-it-works .

Reply:
Sure, will do.